Jellyfin.Api/Controllers/LibraryController.cs

```diff
@@ -309,17 +307,9 @@ public ActionResult<QueryResult<BaseItemDto>> GetCriticReviews()
     [HttpPost("Library/Refresh")]
     [Authorize(Policy = Policies.RequiresElevation)]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
-    public async Task<ActionResult> RefreshLibrary()
+    public ActionResult RefreshLibrary()
     {
-        try
-        {
-            await _libraryManager.ValidateMediaLibrary(new SimpleProgress<double>(), CancellationToken.None).ConfigureAwait(false);
-        }
-        catch (Exception ex)
-        {
-            _logger.LogError(ex, "Error refreshing library");
-        }
-
+        _libraryRefreshManager.StartScan();
```

Comment:
Does this need a change in the UI because `ValidateMediaLibrary` previously seem to be blocking.